libs/ui-components/src/components/ImageBuilds/NewVersionImageBuildModal/NewVersionImageBuildModal.tsx

```
@@ -0,0 +1,139 @@
+          isDisabled={isSubmitting}
+        />
+      </FormGroup>
+      <FormGroup label={t('Destination image tag')} fieldId="textfield-destinationImageTag">
+        <TextField
+          name="destinationImageTag"
+          aria-label={t('Destination image tag')}
+          helperText={t('Tag for the built image. Defaults to the parent tag incremented by one.')}
+          isDisabled={isSubmitting}
+        />
+      </FormGroup>
+      {submitError && <Alert isInline variant="danger" title={t('Failed to create new version')} children={submitError} />}
```

ESLint

[error] Do not pass children as props. Instead, nest children between the opening and closing tags.

(react/no-children-prop)